tuta-sdk/rust/sdk/Cargo.toml

```diff
@@ -7,10 +7,10 @@ version = "262.250122.0"
 async-trait = "0.1.77"
 serde = { version = "1.0.201", features = ["derive"] }
 serde_json = "1.0.117"
-minicbor = { version = "0.24.2", features = ["std", "alloc"] }
-thiserror = "1.0.60"
+minicbor = { version = "0.25", features = ["std", "alloc"] }
+thiserror = "2.0"
 base64 = "0.22.1"
-aes = { version = "0.8.4", features = ["zeroize"] }
+aes = "0.8.4"
```

Comment:
did we lose zeroize?

Reply:
the `aes` crate doesn't have that feature in version 0.8.4

Reply:
I thought they do?
https://docs.rs/crate/aes/0.8.4/features#zeroize

Reply:
https://github.com/RustCrypto/block-ciphers/blob/4da9b802de52a3326fdc74d559caddd57042fed2/aes/src/ni.rs#L168 

and

https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

gah.